### PR TITLE
Use default django queries in addons and tests [OSF-8312]

### PR DESCRIPTION
**Pull request by erinspace**
[#OSF-8312]

## Purpose

Remove usages of modularodm Q and modularodm exceptions from addons!

## Changes

- remove modularodm Q objects
- Remove modularodm exceptions

## Side effects

none anticipated

## Ticket

https://openscience.atlassian.net/browse/OSF-8312

### CODE REVIEW

File: addons/osfstorage/listeners.py [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-
 from website.project.signals import contributor_removed
 
 @contributor_removed.connect
 def checkin_files_by_user(node, user):
     ''' Listens to a contributor being removed to check in all of their files
     '''
     from addons.osfstorage.models import OsfStorageFileNode
-    files = OsfStorageFileNode.find(Q('node', 'eq', node) & Q('checkout', 'eq', user))
+    files = OsfStorageFileNode.objects.filter(node=node, checkout=user)
```
Comment (caseyrollins) [inline addons/osfstorage/listeners.py]: You can use [`update()`](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#update) here and eliminate the loop below. 

Just so you know, `update()` does bypass custom saves, but I checked with Wisecarver that it's perfectly fine to use here.

File: addons/base/models.py [hunk 2]
```diff
@@ -319,7 +318,7 @@ def merge(self, user_settings):
         except KeyError:
             pass
         else:
-            connected = Model.find(Q('user_settings', 'eq', user_settings))
+            connected = Model.objects.filter(user_settings=user_settings)
```
Comment (caseyrollins) [inline addons/base/models.py]: Similar to other note, I think you can use `update()` here and remove the loop below.

File: addons/osfstorage/models.py [hunk 3]
```diff
@@ -104,10 +104,7 @@ def get_file_guids(cls, materialized_path, provider, node=None):
             for item in children:
                 guids.extend(cls.get_file_guids(item.path, provider, node=node))
         else:
-            try:
-                guid = Guid.find(Q('referent', 'eq', file_obj))[0]
-            except IndexError:
-                guid = None
+            guid = Guid.objects.filter(object_id=file_obj.id, content_type=ContentType.objects.get_for_model(file_obj)).first()
```
Comment (caseyrollins) [inline addons/osfstorage/models.py]: Does `file_obj.referents.first()` work here?

**Comment by sloria** [comment]
Merged into release/next-djangole.

**Pull request closed without merging**